Extensions/Leaderboards/JsExtension.js

```diff
@@ -111,6 +111,37 @@ module.exports = {
       .setIncludeFile('Extensions/Leaderboards/leaderboardstools.js')
       .setFunctionName('gdjs.evtTools.leaderboards.formatPlayerName');
 
+    extension
+      .addAction(
+        'DisplayLeaderboard',
+        _('Display leaderboard'),
+        _('Display the specified leaderboard on top of the game. If a leaderboard was already displayed on top of the game, the new leaderboard will replace it.'),
+        _('Display leaderboard _PARAM1_'),
+        '',
+        'JsPlatform/Extensions/leaderboard.svg',
+        'JsPlatform/Extensions/leaderboard.svg'
+      )
+      .addCodeOnlyParameter('currentScene', '')
+      .addParameter('leaderboardId', 'Leaderboard', '', false)
+      .getCodeExtraInformation()
+      .setIncludeFile('Extensions/Leaderboards/leaderboardstools.js')
+      .setFunctionName('gdjs.evtTools.leaderboards.displayLeaderboard');
+
```

Comment:
Let's add a condition "LeaderboardViewErrored" ("The leaderboard view had an error when opening") which returns true when we detect the leaderboard view could not be open, so that the game can react to this :)

Reply:
(see other comments about "loading error handling")